manager/integration/tests/common.py

```diff
@@ -3647,6 +3647,54 @@ def nfs_create_dummy_in_progress_backup():
     raise NotImplementedError
 
 
+def backupstore_create_file(client, core_api, prefix, file_name, data={}):
+    backup_target_setting = client.by_id_setting(SETTING_BACKUP_TARGET)
+    backupstore = backup_target_setting.value
+
+    if is_backupTarget_s3(backupstore):
+        return mino_create_file_in_backupstore(client,
+                                               core_api,
+                                               prefix,
+                                               file_name,
+                                               data)
+
+    elif is_backupTarget_nfs(backupstore):
+        return nfs_create_file_in_backupstore()
+
```

Comment:
```
 else:
    raise Exception()
```

Similar below.

Reply:
Fixed

Reply:
Thanks! I changed it to raise NotImplementedError